typescript/packages/common-identity/src/pass-key.ts

```diff
@@ -31,6 +32,10 @@ export class PassKey {
     this.credentials = credentials;
   }
 
+  id() {
+    return this.credentials.id;
+  }
```

Comment:
Exposed so I can store enough information to reconstruct a key descriptor from local storage

Reply:
This could be `this.credentials.rawId` instead, which is the `ArrayBuffer` form of `id`, which would allow us to get rid of the conversion `Uint8Array.from(atob(storedCredential.id), (c) => c.charCodeAt(0)),` in `utils/credentials.ts`